packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -47,7 +47,7 @@ export const IN_MEETING = 'IN_MEETING';
 export const INCOMING = 'INCOMING';
 export const INTENT_TO_JOIN = 2423005;
 
-
+export const _JOIN_ = 'JOIN';
```

Comment:
i'm not that familiar with how constants are being used in this project, but this and line 52 are pretty confusing to me. a constant with almost identical names and that is differentiated based on casing feels like a recipe for confusion/bugs.

Reply:
We need to refactor the constant file to have more category and meaning full

Reply:
agreed